plugins/Aka/plugin.py

```diff
@@ -376,6 +396,104 @@ def findBiggestAt(alias):
 else:
     raise plugins.NoSuitableDatabase(['sqlite3', 'sqlalchemy'])
 
+
+class AkaHTTPCallback(httpserver.SupyHTTPServerCallback):
+    name = 'Aka web interface'
+    base_template = '''\
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Aka</title>
+    <link rel="stylesheet" href="/default.css">
+  </head>
+
+  <body>
+    <h1>Aka</h1>
+
+    {}
```

Comment:
So Python 2.6 is complaining that this doesn't have an explicit `{0}`, but since it went EOL in 2013, do we need to care?

Reply:
I'll remove its support around 2020-01-01, at the same time as 2.7 and probably 3.2 and 3.3 as well. In the meantime, could you use `%s`, for consistency with other parts of Limnoria?

Reply:
Ah, okay, will do.